provider/datagen/tests/data/postcard/fingerprints.csv

```diff
@@ -4948,7 +4948,7 @@ displaynames/variants@1, th, 3709B, ed1d48e400b1c35c
 displaynames/variants@1, tr, 1094B, f75a0ab8f0d0bc98
 fallback/likelysubtags@1, und, 1963B, ea452a518b2defb6
 fallback/parents@1, und, 2553B, 8f3b5fb7bff27d97
-fallback/supplement/co@1, und, 70B, 71e01f2099eca417
+fallback/supplement/co@1, und, 246B, 78151ac4e67429c5
```

Comment:
is that size increase expected?

Reply:
yes if you look at the json it adds a handful of parent relationships